doc/SUMMARY.md

```diff
@@ -11,5 +11,6 @@
   - [Rayhunter's hueristics](./heuristics.md)
   - [How we analyze a capture](./analyzing-a-capture.md)
 - [Supported devices](./supported-devices.md)
+  - [TP-Link M7350](./tplink-m7350.md)
```

Comment:
I'm a little concerned that at a casual glance someone might miss that we also support the orbic, maybe we should break the orbic out into it's own separate page as well.